overlay/files/system/oem/11_persistency.yaml

```diff
@@ -36,15 +36,21 @@ stages:
           /usr/share/pki/trust/anchors
           /var/lib/ca-certificates
         PERSISTENT_STATE_BIND: "true"
-    - if: |
-        [ -r /run/cos/custom-layout.env ]
+    - if: '[ -f "/run/cos/recovery_mode" ]'
```

Comment:
currently defined here: https://github.com/kairos-io/packages/blob/2ea0ddd0ed30d393b34c1d0356aedb634eba9031/packages/system/cloud-config/oem/00_rootfs.yaml#L16

Reply:
I think actually we have an issue to have all the configs in one place only, part of the #116 epic